pkg/pdutil/api_client.go

```diff
@@ -178,3 +174,33 @@ func (pc *pdAPIClient) listGcServiceSafePoint(
 	}
 	return &resp, nil
 }
+
+// CollectMemberEndpoints return all members' endpoint
+func (pc *pdAPIClient) CollectMemberEndpoints(ctx context.Context) ([]string, error) {
+	members, err := pc.grpcClient.GetAllMembers(ctx)
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+	result := make([]string, 0, len(members))
+	for _, m := range members {
+		result = append(result, m.GetClientUrls()[0])
+	}
+	return result, nil
+}
+
+// Healthy return error if the member corresponding to the endpoint is unhealthy
+func (pc *pdAPIClient) Healthy(ctx context.Context, endpoint string) error {
+	url := endpoint + healthyAPI
+
+	ctx, cancel := context.WithTimeout(ctx, 5*time.Second)
+	defer cancel()
+
+	resp, err := pc.httpClient.Get(ctx, fmt.Sprintf("%s/", url))
+	if err != nil {
+		return errors.Trace(err)
+	}
+
+	_, _ = io.Copy(io.Discard, resp.Body)
```

Comment:
Can we just close body directly and drop its content and instead of `io.Copy(io.Discard, resp.Body)`?